pullRequestHandler.go

```diff
@@ -123,9 +133,97 @@ func hasUnsigned(req types.PullRequestOuter, client *github.Client) (bool, error
 		}
 	}
 
-	return hasUnsigned, err
+	return hasUnsigned
 }
 
 func isSigned(msg string) bool {
 	return strings.Contains(msg, "Signed-off-by:")
 }
+
+func lintCommits(commits []*github.RepositoryCommit) bool {
+	for _, commit := range commits {
+		if commit.Commit != nil && commit.Commit.Message != nil {
+			if lintCommit(commit.Commit.Message) == false {
+				return false
+			}
+		}
+	}
+	return true
+}
+
+func lintCommit(message *string) bool {
+	var valid bool
+
+	if message == nil {
+		return false
```

Comment:
As this is tested in `lintCommits`, is this belt and braces or a hangover?

Reply:
I'm not sure.. I might have to double-check